src/cmds/qsub.c

```diff
@@ -2775,7 +2775,7 @@ fwd_tgt_creds(krb5_context context, krb5_auth_context auth_context, krb5_princip
 	if (!rhost)
 		return ENOMEM;
 	free_rhost = 1;
-	memcpy(rhost, server->data[1].data, server->data[1].length);
+	memcpy(rhost, server->data[1].data, (size_t)server->data[1].length);
```

Comment:
@sid2364 the ticket was only about memmove, so let us just keep it to that for now. However log another bug if the same change is needed in memcpy as well.